backend/src/main/java/org/latexscribe/LatexScribe/webcontroller/DocumentController.java

```diff
@@ -22,6 +22,11 @@ public class DocumentController {
     final private IDocumentService documentService;
     final private ITagService tagService;
 
+    @GetMapping("/")
```

Comment:
```suggestion
    @GetMapping
```

As mentioned in #18 we should remove the prefix